.github/pull_request_template.md

```
@@ -0,0 +1,49 @@
+#### Description (What was done? Why it was done? How it was done?)
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)